chains/v2/types/quartz.json

```diff
@@ -0,0 +1,16 @@
+{
+  "runtime_id": 913010,
+  "types": {
+    "Balance": "u128",
+    "Index": "u64",
```

Comment:
I think it should be 32. Search first AccountInfo in metadata and then search by id of the nonce field's type

Reply:
Ok, I was confused with AccountIndex, now i get it and fix